metadata-service/auth-impl/src/main/java/com/datahub/authentication/token/StatelessTokenService.java

```diff
@@ -95,20 +98,23 @@ public String generateAccessToken(
    * Note that the caller of this method is expected to authorize the action of generating a token.
    */
   @Nonnull
-  public String generateAccessToken(@Nonnull final String sub, @Nonnull final Map<String, Object> claims, final long expiresInMs) {
+  public String generateAccessToken(@Nonnull final String sub, @Nonnull final Map<String, Object> claims, final Optional<Long> expiresInMs) {
```

Comment:
Just across the entire PR, I'd prefer to use a Nullable Long over the additional boxing / unboxing and conditional checks incurred by an Optional parameter